gm4_firework_assemblers/data/firework_assemblers/functions/create.mcfunction

```diff
@@ -0,0 +1,9 @@
+#@s = @e[type=item,nbt=CRAFTING_TABLE] at @s if entity @e[type=item,nbt=FLINT_AND_STEEL,distance=..1] if block ~ ~-1 ~ dropper{CC RECIPE} align xyz positioned ~.5 ~-1 ~.5
```

Comment:
Use more informal worded headers

Reply:
I just copied this from CC. I'll actually change all of my headers to the format I usually use, including what you've just mentioned